config/initializers/devise.rb

```diff
@@ -142,7 +142,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 8..128
+  config.password_length = (Utils.if_present(ENV['MIN_PASSWORD_LENGTH'], :to_i) || 8)..128
```

Comment:
Got it!

On Thu, Mar 17, 2016 at 2:16 AM Dominik Sander <notifications@github.com>
wrote:

> In config/initializers/devise.rb
> <https://github.com/cantino/huginn/pull/1330#discussion_r56474415>:
>
> > @@ -142,7 +142,7 @@
> >
> >    # ==> Configuration for :validatable
> >    # Range for password length.
> > -  config.password_length = 8..128
> > +  config.password_length = (Utils.if_present(ENV['MIN_PASSWORD_LENGTH'], :to_i) || 8)..128
>
> I added the if_present helper to avoid that case. When the argument is
> not present? it returns nil. It does the same as ENV['MIN_PASSWORD_LENGTH'].present?
> ? ENV['MIN_PASSWORD_LENGTH'].to_i : 8, but I did not want to repeat that
> pattern a couple of times.
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/1330/files/df349a73028f623ba9e0dbf68ac6736e1134f2a4#r56474415>
>